samples/bookinfo/routing/route-rule-all-v1-mtls.yaml

```diff
@@ -53,78 +53,3 @@ spec:
         host: details
         subset: v1
 ---
```

Comment:
This file is now the same as route-rule-all-v1.yaml. Delete it?

Reply:
Good catch, removed.